canonical-web-and-design/ubuntu.com
# copy update /openstack/managed

## Done

- Update the page /openstack/managed as requested in the [copy update](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/managed or https://ubuntu-com-10809.demos.haus/openstack/managed
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please mark the discussion as resolved in the [copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#) once the QA is done

## Issue / Card

Fixes #10805


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
